go/mysql/replication_constants.go

```diff
@@ -152,54 +152,82 @@ const (
 // These constants describe the event types.
 // See: http://dev.mysql.com/doc/internals/en/binlog-event-type.html
 const (
-	eUnknownEvent           = 0
-	eStartEventV3           = 1
-	eQueryEvent             = 2
-	eStopEvent              = 3
-	eRotateEvent            = 4
-	eIntVarEvent            = 5
-	eLoadEvent              = 6
-	eSlaveEvent             = 7
-	eCreateFileEvent        = 8
-	eAppendBlockEvent       = 9
-	eExecLoadEvent          = 10
-	eDeleteFileEvent        = 11
-	eNewLoadEvent           = 12
-	eRandEvent              = 13
+	//lint:ignore U1000 unused for now, reserved for future use
```

Comment:
Seems better to me to leave the code as-is and ignore these warnings rather than making the code messy to satisfy an over-eager linter.